### ISSUE TITLE
Error when :ElmFormat on Windows

### ISSUE THREAD

**Issue by kuznero** [ElmCast/elm-vim#77]
I experienced this weird behavior when it does the formatting of the document (`elm-format.exe` is present in the system), but `:ElmFormat` produces following error in Vim:

```
Error detected while processing function elm#Format:
E190: Cannot open "C:\Program Files (x86)\Vim/vimfiles/view/C=-=+Projects=+_kuznero=+vocabularis-elm=+src=+Main.elm=" for writing
```

It should have resolve another target for write, which is typically `%USERPROFILE%\vimfiles\` and not something inside `C:\Program Files....`.

**Comment by jamescleary** [comment]
I was having the same issue. It is a result of bad defaults for Windows versions of Vim. See here, https://groups.google.com/forum/#!topic/vim_dev/5AftH0uBqTs

The fix was adding `set viewdir=$HOME/vimfiles/views/` to my vimrc (or to any directory where you want those files stored).

**Comment by kuznero** [comment]
Thanks a lot! That works. Which leads to this [PR](https://github.com/ElmCast/elm-vim/pull/79).

**Comment by ajhager** [comment]
Fixed by 175af502666fb55f08382d27679a9fdf2a078e8f

**Comment by kuznero** [comment]
Thanks!
